packages/stokhos/src/sacado/kokkos/vector/amesos2/Amesos2_Basker_MP_Vector.hpp

```diff
@@ -68,7 +68,7 @@ namespace Amesos2 {
   // Enable MP::Vector as a valid Scalar type for Basker
   template <class ST>
   struct TypeMap< Basker,Sacado::MP::Vector<ST> > {
-    static Sacado::MP::Vector<ST> dtype;
+    typedef Sacado::MP::Vector<ST> dtype;
```

Comment:
@hkthorn I wanted to check for use of Basker (classic) in Xyce, do you maintain a custom TypeMap struct (similar to this location in Stokhos) and will this change to remove the static type from Basker pose any issues?